src/components/Heroes/HeroListItem.css

```diff
@@ -38,26 +38,27 @@
 }
 
 .HeroItem {
-	min-height: 130px;
+	min-height: 90px;
   cursor: pointer;
   /*min-width: 230px;*/
 	background-size: cover;
 	background-repeat: no-repeat;
 	background-position: center;
 	position: relative;
 	background-color: white;
-	margin: 5px;
+	// margin: 5px;
```

Comment:
then you need to add a margin to the input text in order to be aligned. otherwhise it looks shit.